control/server.py

```diff
@@ -73,7 +68,7 @@ def __exit__(self, exc_type, exc_value, traceback):
             self.logger.info("Terminating SPDK...")
             self.spdk_process.terminate()
             try:
-                timeout = self.config.getfloat("spdk", "timeout")
+                timeout = self.config.spdk.timeout
```

Comment:
The timeout should have a unit: `timeout_seconds` or `timeout_s` or similar also the config parameter itself.